src/gru/gru.py

```diff
@@ -0,0 +1,39 @@
+def gru():
+    import streamlit as st
+    import time
+    from .constants import readme_markdown_text
+    from .helpers import get_qaqc_runs, run_all_workflows
+    from .components import source_table, check_table
+
+    st.header("GRU QAQC")
+    st.write(
```

Comment:
Very nit-picky but you can just use one `st.write` and even more nitpicky...the code goes to triple quoted to single quoted. Obviously this has not impact on the function of the app

Reply:
All for the nitpicks!